apps/captiveportal/qml/pages/CaptivePortalPage.qml

```diff
@@ -104,7 +104,7 @@ Page {
             when: virtualKeyboardObserver.opened && webView.enabled
             PropertyChanges {
                 target: webView.contentItem
-                virtualKeyboardMargin: virtualKeyboardObserver.panelSize
+                virtualKeyboardHeight: virtualKeyboardObserver.imSize
```

Comment:
Need to check in more detail but we should not use the value that is animated -- only one single update and that's it.

Reply:
This is good.